docs/src/prototypes/VirtualizedTable/VirtualizedTable.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  gridCellBehavior,
+  gridHeaderCellBehavior,
+  gridNestedBehavior,
+  gridRowBehavior,
+  Table,
+} from '@fluentui/react'
+import * as React from 'react'
+import { AutoSizer, List as ReactVirtualizedList } from 'react-virtualized'
+import getItems from './itemsGenerator'
+
+const { rows } = getItems()
+
+const VirtualizedTablePrototype = () => {
+  const rowGetter = ({ index }) => {
```

Comment:
```suggestion
  const rowGetter: RowGetter = ({ index }) => {
```

Not sure about type name, please `ListProps` interface

Reply:
Also please move out it to upper scope:

```
const rowGetter = ({ index }) => {}

//

const VirtualizedTablePrototype = () => {
```

Reply:
Moved. but rowGetter doesn't have any typings, so no changes there :(